frontend/src/metabase/components/UserHasSeen/UserHasSeenAllContext.tsx

```diff
@@ -61,7 +61,7 @@ const useUserHasSeenAll = (id: string) => {
     defaultValue: EMPTY_ARRAY,
   });
 
-  useKeyPressEvent("q", () => setSeenBadges([]));
+  useKeyPressEvent("q", () => setSeenBadges && setSeenBadges([]));
```

Comment:
I think it's slightly safer, typewise, to use `setSeenBadges?.([])`

Reply:
didn't know about that syntax, nice!